packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -7,28 +7,31 @@ import MenuItem from '@material-ui/core/MenuItem';
 import { isHideMenuKey, isTabKey } from '../../utils/keyboardUtils';
 import { GridApiContext } from '../GridApiContext';
 import { GridMenu } from '../menu/GridMenu';
-import { GridExportOption } from '../../models';
+import { GridExportOption, GridExportFormat, GridExportConfiguration } from '../../models';
 
-export const GridToolbarExport = React.forwardRef<HTMLButtonElement, ButtonProps>(
-  function GridToolbarExport(props, ref) {
+type GridToolbarExportProps = ButtonProps & { exportConfiguration?: GridExportConfiguration };
+
+export const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportProps>(
+  function GridToolbarExport({ exportConfiguration, ...other }, ref) {
     const apiRef = React.useContext(GridApiContext);
     const exportButtonId = useId();
     const exportMenuId = useId();
     const [anchorEl, setAnchorEl] = React.useState(null);
     const ExportIcon = apiRef!.current.components!.ExportIcon!;
 
-    const ExportOptions: Array<GridExportOption> = [
-      {
-        label: apiRef!.current.getLocaleText('toolbarExportCSV'),
-        format: 'csv',
-      },
-    ];
+    const exportOptions: Array<GridExportOption> = [];
+
+    exportOptions.push({
+      label: apiRef!.current.getLocaleText('toolbarExportCSV'),
+      format: 'csv',
+      options: exportConfiguration?.csv,
+    });
 
     const handleExportSelectorOpen = (event) => setAnchorEl(event.currentTarget);
     const handleExportSelectorClose = () => setAnchorEl(null);
-    const handleExport = (format) => {
-      if (format === 'csv') {
-        apiRef!.current.exportDataAsCsv();
+    const handleExport = (option: GridExportFormat) => {
+      if (option.format === 'csv') {
+        apiRef!.current.exportDataAsCsv(option.options, exportConfiguration?.fileName);
```

Comment:
`option.options` can we avoid redundant path?

Reply:
Word `option` replaced by `variant`. Any other suggestion?